### ISSUE TITLE
[Suggestion] Prioritizing explicit item-usage over OreDict.

### ISSUE THREAD

**Issue by ZeroLevels** [mezz/JustEnoughItems#1894]
If a specific item is used in a recipe and **not any of its oredict entries**, then I feel checking usage would benefit from showing those explicit usages first.  No real need for a separate UI Page. Oredict usages would still show, just afterward.

**Comment by jursamaj** [comment]
Yes, yes, yes!  So many times, I've wanted to know if some item is worth holding on to for a special usage, rather than being a trivial variation of something I already have plenty of.  I'd think the easiest implementation would be a modifier key held when pressing U or right-clicking.

As a related issue, it would be nice if oredicted items in recipes had some indicator that they *are* oredicted, beyond the cycling.

**Comment by mezz** [comment]
Thanks for the suggestion!

This was implemented in JEI 30.20.0 for Minecraft 26.2 here:
* https://github.com/mezz/JustEnoughItems/commit/2e90070fb591208b6102c969a9867891b93bb8ae

Recipes that use the exact ingredient are now sorted before recipes that accept it through a tag or other grouped ingredient.
